components/config-provider/index.en-US.md

```diff
@@ -0,0 +1,30 @@
+---
+category: Components
+type: Other
+cols: 1
+title: ConfigProvider
+---
+
+`ConfigProvider` provides a uniform configuration support for components.
+
+## Usage
+
+`ConfigProvider` takes use of [context](https://facebook.github.io/react/docs/context.html), a feature of React, to accomplish global effectiveness by wrapping the app only once.
```

Comment:
This component provides a configuration to all React components underneath itself via the [context API](https://facebook.github.io/react/docs/context.html), In the render tree all components will have access to the provided config.